setup.cfg

```diff
@@ -0,0 +1,49 @@
+[metadata]
+name = jina
+version = 3.11.1
```

Comment:
we need to adapt our CI/CD to update this `version` then

Reply:
yup

Reply:
we can do this later. I think the current change of just moving the build configs is enough